rust_src/src/symbols.rs

```diff
@@ -212,6 +214,32 @@ impl LispSymbolRef {
         blv.where_ = Qnil;
         blv.set_found(false);
     }
+
+    pub fn default_toplevel_binding_rust(&self) -> SpecbindingRef {
+        let current_thread = ThreadState::current_thread();
+        let specpdl = SpecbindingRef::new(current_thread.m_specpdl);
+
+        let mut binding = SpecbindingRef::new(std::ptr::null_mut());
+        let mut pdl = SpecbindingRef::new(current_thread.m_specpdl_ptr);
+
+        while pdl > specpdl {
+            unsafe {
+                pdl.ptr_sub(1);
```

Comment:
where is `ptr_sub` coming from? I know of `ptr::sub`. That does not change the pointer but instead returns a new one. So it would be `pdl = pdl.sub(1);`

Reply:
That's coming from a method on `ExternalPtr`, it's the one(s) for modifying the pointer I added.

Reply:
Thx @agraven for explanation. I believe this should be okay then.